src/helpers/misc/AnimalActions.re

```diff
@@ -244,3 +247,50 @@ let useWithdrawDeposit = () => {
     txState,
   );
 };
+let useChangePrice = animal => {
```

Comment:
This function is basically a copy/paste of the functions for buy/withdraw/deposit above.